src/components/routes/Missions.module.css

```diff
@@ -0,0 +1,43 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+body {
+  font-family: 'Roboto', sans-serif;
+  line-height: 1.5;
+}
+
+table {
+  margin: auto;
+  width: 90%;
+  margin-top: 30px;
+}
+
+table,
+th,
+td {
+  border: 1px solid #ccc;
+  border-collapse: collapse;
+  padding: 20px;
+}
+
+tbody > tr:nth-child(odd) {
+  background-color: #f4f4f4;
+}
```

Comment:
Using element Selector in CSS modules would effect styles of other components.

Kindly use class or id Selector to prevent side effect.